metadata-ingestion/examples/recipes/hana_to_datahub.yml

```diff
@@ -0,0 +1,21 @@
+---
+# see https://datahubproject.io/docs/metadata-ingestion/source_docs/hana for complete documentation
+source:
+  type: "hana"
+  config:
+    host_port: localhost:39041
+    database: HXE
+    username: SYSTEM
+    password: HXEHana1
```

Comment:
Please don't put dummy passwords as example. This is bad practice that we are trying to remove. Please add something like "${HANA_PASS}"